webdrops/login.go

```diff
@@ -37,13 +38,15 @@ func parseTemplate(tmplS string, args interface{}) (io.Reader, error) {
 }
 
 func (sess *Session) Login() error {
-	payload, err := parseTemplate(
```

Comment:
I don't know why I implemented it that way, it was totally awful.
I'll add a commit where i delete the `parseTemplate` func altogether...